usaspending_api/search/models/award_search.py

```diff
@@ -13,6 +13,8 @@ class AwardSearch(models.Model):
     category = models.TextField(null=True, db_index=True)
     type = models.TextField(null=True, db_index=True)
     type_description = models.TextField(null=True)
+    type_raw = models.TextField(null=True, db_index=True)
+    type_description_raw = models.TextField(null=True)
```

Comment:
Shouldn't have much of an impact, but it would be nice to see the column order of these two Django models match the Delta models (raw type fields appear before type fields)